docs/reference/files/charmcraft-yaml-file.rst

```diff
@@ -504,7 +504,7 @@ for the OCI image resource used to create the container; to use it, specify  an
 image resource name (that you will then define further in the `resources`_ block).
 ``bases`` is a list of bases to be used for resolving a container image, in descending
 order of preference; to use it, specify a base name (for example, ``ubuntu``,
-``centos``, ``osx``, ``opensuse``), a
+``almalinux``), a
```

Comment:
what about `osx` and `opensuse`?

Reply:
They're not supported bases - IDK where the values there originate.